cluster/manifests/coredns-local/daemonset-coredns.yaml

```diff
@@ -51,6 +51,18 @@ spec:
         - containerPort: 53
           name: dns-tcp
           protocol: TCP
+{{- if eq .Cluster.ConfigItems.dns_unbound_liveness_probe "true" }}
+        livenessProbe:
+          exec:
+            command:
+            - dig
+            - "+short"
+            - "@127.0.0.1"
+            - "kubernetes.default.svc.cluster.local"
+          initialDelaySeconds: 5
```

Comment:
Let's increase this a bit? This depends on CoreDNS being up, which can take more than 5 seconds if it needs to be pulled. It's also very unlikely that it'll get stuck in the first 5 seconds.

Reply:
Increased to 60, similar to coredns livenessProbe